django/test/client.py

```diff
@@ -690,7 +690,10 @@ def logout(self):
 
     def _parse_json(self, response, **extra):
         if not hasattr(response, '_json'):
-            if 'application/json' not in response.get('Content-Type'):
+            # Use regex to support JSON Vendor Tree specification
+            # https://tools.ietf.org/html/rfc6838#section-3.2
+            json_content_type_re = re.compile(r'^application\/(vnd\..+\+)?json$')
```

Comment:
Would be better to put this as a module constant to avoid compiling it over and over.